### DISCUSSION TITLE
Actividad 1-GRUPO 7

### DISCUSSION THREAD

**Discussion by IZelayeta** [frlp-utn-ingsoft/catedra#99] [Show and tell]
https://github.com/IZelayeta/vetsoft

**Comment by ivanduva** [comment]
Buenas,
La actividad está muy bien resuelta. Queda aprobada.
Además valoramos que hayan hecho opcionales y que hayan estandarizado los commits, si bien se puede mejorar.

Les dejamos algunos comentarios que pueden servir para las próximas actividades:

- Vimos que en varios casos hicieron las migraciones casi al finalizar la funcionalidad. Tengan cuidado con esto porque eso significa que en los commits anteriores la app no funciona. Recuerden hacer las migraciones cada vez que modifican el modelo y de probar la app cada vez que modifican algo para comprobar que funciona.
- En algunos casos vimos mensajes de commits haciendo referencia a archivos o funciones. Esto es redundante porque el commit ya tiene esa información. El mensaje debería contener una síntesis de la funcionalidad que resuelve el commit.
- Si bien no es el objetivo del trabajo, se puede mejorar la UI agregando accesos directos a cada tabla desde la navbar o agregando un mensaje de error en la validación del form de proveedor para el email. Estos son algunos ejemplos, pero lo que queremos decir es que alentamos a que sigan mejorando la app por fuera de lo obligatorio siempre que quieran y puedan porque lo que hicieron quedó muy bien.

Buen trabajo!
Saludos
